Formula/tg.rb

```diff
@@ -0,0 +1,28 @@
+class Tg < Formula
+  include Language::Python::Virtualenv
+
+  desc "Terminal telegram client"
+  homepage "https://github.com/paul-nameless/tg"
+  url "https://github.com/paul-nameless/tg/archive/refs/tags/v0.11.0.tar.gz"
+  sha256 "2303983af7b7246fb3198bcae688d3b6327a96bbfc91c94ef9bc783cc7191f42"
+  license "Unlicense"
+
+  bottle :unneeded
+
+  depends_on "python@3.9"
+  depends_on "tdlib" => "1.7.0"
+
+  resource "python-telegram-patched" do
+    url "https://github.com/paul-nameless/python-telegram/archive/refs/tags/0.14.0-patched.tar.gz"
```

Comment:
Do you plan to maintain this fork of `python-telegram` indefinitely? It's probably better to try to upstream your changes in some form here, no?

Reply:
There is no other way that I am aware of to fix this issue: https://github.com/Homebrew/discussions/discussions/1336#discussioncomment-645775

Cause that package contains tdlib which is not needed as I am installing tdlib with formula but otherwise brew audit fails. 
So if there is any other solutions I will be happy to know them.

This requirement isn't updated very often, like once a year so I am indeed planning to support this package.

Reply:
Make a pull request upstream to (optionally) stop shipping their own tdlib.

Reply:
there is an issue about this and author doesn't want to remove libs.
https://github.com/alexander-akhmetov/python-telegram/issues/119

Reply:
I'm reading that issue, but it doesn't give me the impression that the author doesn't want to remove the libs. It seems they're happy to find a solution that works for packagers, but just haven't found one yet.